src/xenia/base/memory.h

```diff
@@ -724,11 +724,17 @@ static void ReadLineNT(CacheLine* XE_RESTRICT destination,
                        const CacheLine* XE_RESTRICT source) {
   assert_true((reinterpret_cast<uintptr_t>(source) & 63ULL) == 0);
 
+#ifdef XENIA_PLATFORM_WINDOWS
```

Comment:
I think that this whole ReadLineNT and WriteLineNT should be in memory_win and there should be separate implementations for "AVX copying"

Reply:
I'm not sure if I understand exactly how I should change this, so I'll probably leave this to someone else to fix.